static/src/stylesheets/module/_layout-hints.garnett.scss

```diff
@@ -0,0 +1,187 @@
+@mixin deport-left {
+    @include mq(leftCol) {
+        margin-left: -1 * ($left-column + $gs-gutter);
+    }
+
+    @include mq(wide) {
+        margin-left: -1 * ($left-column-wide + $gs-gutter);
+    }
+}
+
+@mixin reduce-left {
+    @include mq(leftCol) {
+        width: $left-column;
+    }
+
+    @include mq(wide) {
+        width: $left-column-wide;
+    }
+}
+
+@mixin pull-quote {
+    @include fs-headline(4);
+    float: left;
+    margin-top: 0;
+    margin-right: 24px;
+    font-weight: 200;
+    font-style: normal;
+    color: $neutral-2;
+
+    @if ($svg-support) {
+        &:before {
+            @extend %i-quote-blue;
```

Comment:
I am not sure if these are referring to anything. Can we remove them?

Reply:
Good spot. Removed.